plugins/module_utils/network/sonic/argspec/bgp_af/bgp_af.py

```diff
@@ -71,6 +71,13 @@ def __init__(self, **kwargs):
                                     'required': True,
                                     'type': 'str'
                                 },
+                                'aggregate_address_config': {
+                                    'elements': 'dict',
+                                    'options': {
```

Comment:
In addition to 'prefix', the following additional (optional) suboptions are needed:

-     as_set (bool; operationally defaulted to False)
-     summary_only (bool; operationally defaulted to False)
-     route_map (str)